src/index.js

```diff
@@ -1,20 +1,35 @@
-import _ from 'lodash';
-import printMe from './print.js';
 import './style.css';
 
-function component() {
-	const element = document.createElement('div');
-	const btn = document.createElement('button');
-  
-	element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-	element.classList.add('hello');
-
-	btn.innerHTML = 'Click me and check the console!';
-	btn.onclick = printMe;
-
-	element.appendChild(btn);
-  
-	return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+window.addEventListener('load', () => {
+  const form = document.querySelector('.add-list');
+  const input = document.querySelector('#item-add');
+  const listEl = document.querySelector('.taskContainer');
+
```

Comment:
- [ ] Kindly create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object should contain three keys, to complete the list dynamically:

- description [string].
- completed [bool].
- index: [number].